### PR TITLE
Add logic for three different legal versions of norent letter content

### PR DESCRIPTION
**Pull request by sraby**
This PR configures our NoRent letter content to fit three different versions, configured in the national metadata and determined by how strong the state's legal protections are. It also updates content in the email and preview on the landing page.

### CODE REVIEW

**Comment by toolness** [comment]
Oy, can we use a term other than "versioned" for this?  Because we're also working on a software product, when I see "versioned letter content" I think that means that we've got different git revisions of letter content or something, which this isn't about.

What about "legal versions" instead of just "versions"?

File: frontend/lib/norent/letter-content.tsx [hunk 1]
```diff
@@ -168,34 +166,66 @@ export const NorentLetterEmailForUserStaticPage: React.FC<{}> = () => (
...
+    .whichVersion;
+  return (
+    <>
+      <LetterTitle {...props} />
+      <p className="has-text-right">{friendlyDate(new Date())}</p>
+      <LetterHeading {...props} />
+      <p>
+        Dear <LandlordName {...props} />,
+      </p>
+      {letterVersion === CovidStateLawVersion.V1_NON_PAYMENT ? (
```
Comment (toolness) [inline frontend/lib/norent/letter-content.tsx:181]: This is sort of type nerdery, but if you move the version-specific content into a separate function that uses a `switch` statement, you can use TypeScript's [exhaustiveness checking](https://www.typescriptlang.org/docs/handbook/advanced-types.html#exhaustiveness-checking) so that if we ever decide to add another "legal version type" to our Enum, the source code will fail to typecheck because we haven't dealt with the new version in our switch statement!  It's kinda neat in that u can make sure you're dealing with all the edge cases without having to even run the test suite.

Aannnnyhoo that's obviously for another day when we have a lot more time on our hands!

Comment (sraby) [reply frontend/lib/norent/letter-content.tsx:181]: hahahah definitely. Tbh I hate switch statements... they're so syntactically verbose!
